lms/envs/aws.py

```diff
@@ -663,6 +663,7 @@
 
 # Student identity verification settings
 VERIFY_STUDENT = AUTH_TOKENS.get("VERIFY_STUDENT", VERIFY_STUDENT)
+ENABLE_HOUSTON_PHOTO_VERIFICATIONS = ENV_TOKENS.get("ENABLE_HOUSTON_PHOTO_VERIFICATIONS", ENABLE_HOUSTON_PHOTO_VERIFICATIONS)
```

Comment:
oe, can we please move this to the FEATURE set, this is a feature flag and not a global thing flag :)

Reply:
Done, defaults to false now.